srtcore/epoll.h

```diff
@@ -115,6 +115,22 @@ struct CEPollDesc
        }
 
        int edgeOnly() { return edge & watch; }
+
+       bool clear(int direction)
```

Comment:
Some doxygen-styled description might be helpful, while you remember the purpose.
```
/// Clear all flags for...
```

Reply:
Right. :)